config/hardware/extruder/TMC/e_TMC2240.cfg

```diff
@@ -3,32 +3,12 @@
 variable_etmc_driver: "TMC2240"
 gcode:
 
-
-## Based upon the settings provided by Esoterical
-## https://github.com/Esoterical/voron_canbus/blob/main/toolhead_flashing/common_hardware/BigTreeTech%20SB2209%20and%20SB2240/SB2240%20Stepper%20Configuration%20and%20Undervoltage.md
 [tmc2240 extruder]
 cs_pin: E_CS
-spi_speed: 500000
 spi_software_sclk_pin: E_SCLK
 spi_software_mosi_pin: E_MOSI
 spi_software_miso_pin: E_MISO
-interpolate: False
 run_current: 0.6
-rref: 12000
 stealthchop_threshold: 99999
```

Comment:
Can you move this line also? I feel that stealchop should not be enablde by default